src/lib/FocusTrap.ts

```diff
@@ -0,0 +1,137 @@
+import { decodeKeydown } from './util';
+
+const FOCUSABLE_ELEMENTS = [
```

Comment:
Do we need to check for any other attributes like `tabindex="-1"` or `aria-disabled`?

Reply:
`tabindex="-1"` doesn't seem necessary since those elements wouldn't be keyboard navigable. `aria-disabled` also doesn't seem relevant because the control is disabled anyway and can't be interacted with